lib/manageiq/providers/kubernetes/debug_kubeclient_watch.rb

```diff
@@ -0,0 +1,13 @@
+require 'kubeclient'
+
+module ManageIQ
+  module Providers
+    module Kubernetes
+      module DebugKubeclientWatch
+        def build_client
+          super.use(logging: { logger: Logger.new($stderr) })
```

Comment:
would a `tap` fix this?

i.e.:

```suggestion
          super.tap { |client| client.use(:logging => {:logger => Logger.new($stderr) }) }
```

... sorry. flyby ...